README.md

```diff
@@ -72,14 +72,16 @@
 
 - **[Creating a data model]**
 - **[Process data in model]**
+- **[Added Validations in model]**
+- **[Added Unit Tests in model]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo]()
+- [Live Demo](CommingSoon)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] @Metaverse-Nabeel Grea work on your documentation however you added the wrong command to run the app, and run the tests for this app , also the deployment section is empty, kindly consider rectifying the errors in the getting started section.
![image](https://github.com/Metaverse-Nabeel/blog_app/assets/61408860/657bea12-ea81-45bf-bf7f-280449c7f2c0)